kratos/processes/check_skin_process.h

```diff
@@ -139,7 +104,7 @@ class CheckSkinProcess: public Process
         if(mrModelPart.Conditions().size() == 0 && mrModelPart.Elements().size() != 0)
             KRATOS_THROW_ERROR(std::invalid_argument, "the number of conditions is zero and the number of elements is not, hence the skin can not envelope the domain","")
 
-        typedef boost::unordered_map<vector<unsigned int>, unsigned int, KeyHasher, KeyComparor > hashmap;
+        typedef std::unordered_map<vector<unsigned int>, unsigned int, KeyHasherRange<vector<unsigned int>>, KeyComparorRange<vector<unsigned int>> > hashmap;
```

Comment:
This is boost vector equivalent to std::vector. I haven't change to std::vector because you told me that we will have our own implementation mimicking the boost behavior

Reply:
Your are right DenseVector is the correct one 
    
       template <typename TDataType> using DenseVector = boost::numeric::ublas::vector<TDataType>;